app/helpers/application_helper.rb

```diff
@@ -53,4 +53,49 @@ def render_index_grid(grid)
     no_records = grid.current_page_records.length < 1
     render "admin/common/grid", no_records: no_records, grid: grid
   end
+
```

Comment:
This is great stuff but it feels wrong to put it in application_helper. How about a new helper file? "old_system_helper"? Or maybe it even belongs in its own class? Thoughts? I bet @hooverlunch has some, if he wants to comment.

In either case, I'd make the transaction type ids and the base_uri into constants.